src/platforms/kotlin-multiplatform/dsym.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Upload Debug Symbols
```

Comment:
The file is called `dsym` but it's generic to debug symbols for Android/iOS, I'd rather rename the file.